scripts/update/sudoers.sh

```diff
@@ -48,3 +49,8 @@ www-data     ALL = (ALL) NOPASSWD: CLEANMEM, SYSCMNDS, GENERALCMNDS
 SUD
     fi
 fi
+
+if ! groups "$master" | grep -q sudo; then
+    usermod -a -G sudo "$master"
+    # TODO remove the sudoers file? what about if this runs again, and the master is present in the userlist?
```

Comment:
Should we change this?

Reply:
"this" was meant to be the entire updater script

Reply:
May as well if we are moving setup that direction